client.py

```diff
@@ -1,95 +1,130 @@
+
+def country(code: str, name: str) -> HazelcastJsonValue:
+    return HazelcastJsonValue({
+        "isoCode": code,
+        "country": name,
+    })
 
 
-def sql_example(hz_client):
+def map_example(client: hazelcast.HazelcastClient):
+    """
+    This example shows how to work with Hazelcast maps.
+    """
+    cities = client.get_map("cities").blocking()
+    cities.put("1", city("United Kingdom", "London", 9_540_576))
+    cities.put("2", city("United Kingdom", "Manchester", 2_770_434))
+    cities.put("3", city("United States", "New York", 19_223_191))
+    cities.put("4", city("United States", "Los Angeles", 3_985_520))
+    cities.put("5", city("Turkey", "Ankara", 5_309_690))
+    cities.put("6", city("Turkey", "Istanbul", 15_636_243))
+    cities.put("7", city("Brazil", "Sao Paulo", 22_429_800))
+    cities.put("8", city("Brazil", "Rio de Janeiro", 13_634_274))
+    map_size = cities.size()
+    print(f"'cities' map now contains {map_size} entries.")
+    print("-"*20)
+
+
+def sql_example(client: hazelcast.HazelcastClient):
+    svc = client.sql
+    create_mapping_for_capitals(svc)
+    clear_capitals(svc)
+    populate_capitals(svc)
+    select_all_capitals(svc)
+    select_capital_names(svc)
+
+
+def create_mapping_for_capitals(sql_service: hazelcast.client.SqlService):
     print("Creating a mapping...")
-    # See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
-    mapping_query = "CREATE OR REPLACE MAPPING cities TYPE IMap " \
-                    "OPTIONS ('keyFormat'='varchar','valueFormat'='varchar')"
-    hz_client.sql.execute(mapping_query).result()
-    print("The mapping has been created successfully.")
-    print("--------------------")
+    # See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
+    mappingQuery = '''
+    CREATE OR REPLACE MAPPING capitals
+        TYPE IMap
+        OPTIONS (
+            'keyFormat' = 'varchar',
+            'valueFormat' = 'varchar'
+        )
+    '''
+    sql_service.execute(mappingQuery).result()
+    print("-" * 20)
+
 
+def clear_capitals(sql_service: hazelcast.client.SqlService):
     print("Deleting data via SQL...")
-    delete_query = "DELETE FROM cities"
-    hz_client.sql.execute(delete_query).result()
+    sql_service.execute("DELETE FROM capitals").result()
     print("The data has been deleted successfully.")
-    print("--------------------")
+    print("-" * 20)
 
-    print("Inserting data via SQL...")
-    insert_query = """
-    INSERT INTO cities VALUES
-    ('Australia','Canberra'),
-    ('Croatia','Zagreb'),
-    ('Czech Republic','Prague'),
-    ('England','London'),
-    ('Turkey','Ankara'),
-    ('United States','Washington, DC');
-    """
-    hz_client.sql.execute(insert_query).result()
-    print("The data has been inserted successfully.")
-    print("--------------------")
 
+def populate_capitals(sql_service: hazelcast.client.SqlService):
+    print("Inserting data via SQL...")
+    insertQuery = '''
```

Comment:
should be snake case

Reply:
Fixed at: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/9/commits/62b8e5462a629de929e28272260d2ea6365239c4